Makefile

```diff
@@ -51,10 +51,33 @@ fix-all:
 	make black-fix
 	make lint-fix
 
+# Download binaries from Taskcluster to run tests
+download-toolchain:
+	BIN=bin bash utils/download-toolchain.sh
+
 # Run unit tests
+# Some tests work only on Linux, use Docker if running locally on other OS
 run-tests:
 	poetry install --only tests --only utils --no-root
-	PYTHONPATH=$$(pwd) poetry run pytest tests -vv
+	PYTHONPATH=$$(pwd) poetry run pytest -vv
+
+# Run unit tests locally under Docker
+# !!! IMPORTANT !!! on Apple Silicon run without poetry shell for the first time
+# as it can change `uname -m` output to x86_64 if it runs under Rosetta
+run-tests-docker:
```

Comment:
I'm running into issues getting docker working locally. This would benefit to break out into a separate PR and land first.